target/snitch_cluster/sw/apps/snax-block-gemm/src/snax-gemm-params.h

```diff
@@ -0,0 +1,8 @@
+#ifndef GEMM_PARAMETER_H
+#define GEMM_PARAMETER_H
+
+#define tileSize 8
+#define meshRow 8
+#define meshCol 8
+
+#endif
```

Comment:
missing a newline, you can set a setting in vscode that always inserts a newline at the bottom